patches/ui-base-BUILD.gn.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
-index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..25349fd0787e0fd9c913fab16d3c646bf856f1f9 100644
+index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..3fbf1a1e7276481638ad79943b9e4cfa51ed6698 100644
 --- a/ui/base/BUILD.gn
 +++ b/ui/base/BUILD.gn
 @@ -395,6 +395,7 @@ jumbo_component("base") {
```

Comment:
I don't think this diff is needed, since `//brave/ui/base:chromium_deps` also includes `//brave/ui/webui/resources`.

Reply:
Updated!

Reply:
I'm confused - we _do_ need this patch otherwise nothing is including `brave/base/ui/BUILD.gn`

Reply:
Right, what I meant is we need the patch, but the patch has 2 diffs in it. I don't think the first diff is needed.

Reply:
I can fix it when I merge C74 in, though, so no big deal. It doesn't hurt, it's just redundant.

Reply:
ah - this didn't change... it's just part of the diff actually

Reply:
From what I see Pete's original change removes that part: https://github.com/brave/brave-core/pull/2036/files#diff-588f3fba9a54e71be0b127f12f6faf94L9